src/frontend/src/session.rs

```diff
@@ -1120,6 +1120,24 @@ fn infer(bound: Option<BoundStatement>, stmt: Statement) -> Result<Vec<PgFieldDe
                         DataType::Varchar.type_len(),
                     ),
                 ])
+            } else if name.eq_ignore_ascii_case("PARAMETERS") {
+                Ok(vec![
+                    PgFieldDescriptor::new(
+                        "Name".to_string(),
+                        DataType::Varchar.to_oid(),
+                        DataType::Varchar.type_len(),
+                    ),
+                    PgFieldDescriptor::new(
+                        "Value".to_string(),
+                        DataType::Varchar.to_oid(),
+                        DataType::Varchar.type_len(),
+                    ),
+                    PgFieldDescriptor::new(
+                        "Mutable".to_string(),
+                        DataType::Varchar.to_oid(),
+                        DataType::Varchar.type_len(),
+                    ),
```

Comment:
`Mutable` sounds like a boolean, so I looked it up:
https://github.com/risingwavelabs/risingwave/blob/23b69c47290a29267694a1ea362b848faaa792d6/src/frontend/src/handler/variable.rs#L193-L197

Maybe we should refactor and share the same `Vec<PgFieldDescriptor>` between these 2 places.